demo/admin/src/formBuilder/FormBuilderAddForm.cometGen_disabled.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * TODO: Re-enable this file when the missing `block` value in `output` can be handled by the generator
```

Comment:
Do we have a task for this?

Reply:
Yes: https://vivid-planet.atlassian.net/browse/COM-1560